.tekton/determine-image-tag-task-oci-ta.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: determine-image-tag-oci-ta
+  namespace: rh-acs-tenant
+# TODO(ROX-23812): Refactor to a task bundle
+spec:
+  description: Determines the tag for the output image using the StackRox convention from 'make tag' output.
+  params:
+  - name: tag-suffix
+    description: Suffix to append to generated image tag.
+    type: string
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the artifact with
+      the application source code.
+    type: string
+  results:
+  - name: image-tag
+    description: Image Tag determined by custom logic.
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  stepTemplate:
+    volumeMounts:
+      - mountPath: /var/workdir
+        name: workdir
+  steps:
+  - name: use-trusted-artifact
```

Renaming suggestion. WDYT of
```suggestion
  - name: get-source-trusted-artifact
```
?

---

I would prefer `use-trusted-artifact` for consistency with Konflux tasks: https://github.com/konflux-ci/build-definitions/blob/main/task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml#L97-L101